src/xfastertransformer/__init__.py

```diff
@@ -23,13 +23,13 @@
 
 _import_structure = {
     "automodel": ["AutoModel"],
-    "tools": ["LlamaConvert", "ChatGLMConvert", "ChatGLM2Convert", "OPTConvert", "BaichuanConvert"],
+    "tools": ["LlamaConvert", "ChatGLMConvert", "ChatGLM23Convert", "OPTConvert", "BaichuanConvert"],
```

Comment:
I think providing three API `ChatGLM2Convert`, `ChatGLM3Convert`,`ChatGLM23Convert` is more friendly for user and forward compatible, even they are one thing.

Reply:
> the suggestion given by pujiang is not to duplicate the code, just modify the name to quickly support it.

No need to duplicate the code...
`ChatGLM23Convert` as the parent classes, set name `chatglm23` in config.ini.
The others are child class. different is the __init__().

Reply:
a new model type in config.ini need duplicate src/models/x.cpp, it is not a passable parameter.

Reply:
> a new model type in config.ini need duplicate src/models/x.cpp, it is not a passable parameter.

If you don't want to inherit multiple classes in C++, why can't the higher-level Python provide three api? As long as the generated config file is the same.